### ISSUE TITLE
[BUG] Dimmed photos/videos

### ISSUE THREAD

**Issue by andrzej-r** [giejay/Immich-Android-TV#57]
Photo/video views appear dimmer than usual (difficult to say for sure as I don't have a side-by-side comparison). The brightness is similar to that of other applications displaying UI elements (usually the main media view gets dimmer when the user interacts with the app). Immich app appears to use such a dimmer/darker view all the time.

**My setup:**
 - Android OS version 9
 - Device: Sony Bravia

**Comment by giejay** [comment]
I have the same TV but can't say that my photos/videos are dim... I'm also not doing anything regarding brightness in the code.

**Comment by andrzej-r** [comment]
Yes, I'm pretty sure that's not the app itself. It is also quite difficult to quantify the "dimming". It feels fairly obvious when switching from another app but I get used to it after a minute or two.

If it is not some kind of Android API automatically dimming media contents because it thinks the user is interacting with the app, then maybe something to do with HDR?

**Comment by andrzej-r** [comment]
I've got some info that may make it possible to reproduce the issue. When in the Immich app, when I open TV's Picture Settings, some of the picture adjustments have no effect on the displayed image. Brightness and peak luminance work as expected, but contrast, black level or gamma have no effect. In other applications all settings have an effect on the displayed contents.

When I max out brightness and peak luminance, the result is ~OK but then other apps are a bit too bright. It looks like Immich app may be assuming a minimum value of contrast and/or gamma.

**Comment by mailinglists35** [comment]
this issue persists. HDR10 HEVC videos created with Samsung S23 phone are darker.

perhaps you could take a look at how Jellyfin solved the problem, as they manage to produce full brightness for the same clip?

https://github.com/jellyfin/jellyfin-androidtv

**Comment by mailinglists35** [comment]
> Im sorry but I have no idea how to fix this and also can't reproduce it

is very easy to reproduce. I have uploaded MP4 sample. watch it on a HDR capable device and see the logcat difference between jellyfin and immich tv. https://www.pastefile.com/ \<remove this\> / mds1m9

**Comment by mailinglists35** [comment]
@giejay  see above ;-)

**Comment by mailinglists35** [comment]
@giejay ping - you might have mised the uploaded sample and comments above

**Comment by giejay** [comment]
No didn't miss it;) My holiday is over so don't have a lot of spare time anymore and I'm also still working on another feature so this will not get priority. If you could raise a PR, possibly with help of AI, that would be great.

**Comment by mailinglists35** [comment]
oh that's sad news, as HDR video is mainstream already, at least those produced by flagship smartphones.
would you reopen the issue though?
